packages/llm-bot/test/fixtures/UmaEcosystem.Fixture.ts

```diff
@@ -0,0 +1,79 @@
+import { addGlobalHardhatTestingAddress, ZERO_ADDRESS } from "@uma/common";
+import {
+  AddressWhitelistEthers,
+  FinderEthers,
+  IdentifierWhitelistEthers,
+  MockOracleAncillaryEthers,
+  RegistryEthers,
+  StoreEthers,
+  TimerEthers,
+  VotingTokenEthers,
+} from "@uma/contracts-node";
+import { zeroRawValue } from "../constants";
+import { formatBytes32String, getContractFactory, hre, Signer } from "../utils";
+import { utils } from "ethers";
+
+export interface UmaEcosystemContracts {
+  finder: FinderEthers;
+  collateralWhitelist: AddressWhitelistEthers;
+  identifierWhitelist: IdentifierWhitelistEthers;
+  registry: RegistryEthers;
+  store: StoreEthers;
+  votingToken: VotingTokenEthers;
+  mockOracle: MockOracleAncillaryEthers;
+  timer: TimerEthers;
+}
+
+export const umaEcosystemFixture = hre.deployments.createFixture(
```

Comment:
this was a lot of work just to set up the testing! is this something we would want to reuse?